public/locales/en.json

```diff
@@ -2,52 +2,52 @@
   "blocks/ArticleSlideshow/ArticleSlideshow.js": {
     "Read more": ""
   },
-  "containers/App/partials/AppCartDrawer.js": {
+  "containers/Cart/Cart.js": {
+    "Empty cart": ""
+  },
+  "containers/CartItem/CartItem.js": {
+    "Decrease quantity to ${0}": "",
+    "Increase quantity to ${0}": "",
+    "Quantity": "",
+    "Remove": "",
+    "Size": ""
+  },
+  "containers/CartSummary/CartSummary.js": {
+    "Shipping": "",
+    "Total": ""
+  },
+  "layouts/App/partials/AppCartDrawer.js": {
```

Comment:
I wonder if we would benefit in moving all these declarations into a separate files.

So we would've have the following structure:
```
i18n/
├─ containers.js
├─ layouts.js
├─ blocks.js
```

and then in, for example, `containers/`:

```json
  "Cart/Cart.js": {
    "Empty cart": ""
  },
  "CartItem/CartItem.js": {
    "Decrease quantity to ${0}": "",
    "Increase quantity to ${0}": "",
    "Quantity": "",
    "Remove": "",
    "Size": ""
  },
  "CartSummary/CartSummary.js": {
    "Shipping": "",
    "Total": ""
  },
```

The reasons why i think this would be beneficial are:
1. It's easier to sort
2. It's easier to have multiple PR's open regarding i18n changes.

What do you think? :)